specification/error-handling.md

```diff
@@ -52,7 +52,50 @@ Another example may be a metric exposed by a `SpanProcessor` that describes the
 Whenever the library suppresses an error that would otherwise have been exposed to the user, the library SHOULD log the error using language-specific conventions.
 SDKs MAY expose callbacks to allow end users to handle self-diagnostics separately from application code.
 
-## Exceptions to the rule
+## Configuring Error Handlers
 
-SDK authors MAY supply settings that allow end users to change the library's default error handling behavior.
+SDK authors MUST supply settings that allow end users to change the library's default error handling behavior.
 Application developers may want to run with strict error handling in a staging environment to catch invalid uses of the API, or malformed config.
+Note that configuring a custom error handler in this way is the only exception to the basic error handling principles outlined above. 
+
+The mechanism by which end users set or register a custom error handler should follow language-specific conventions. 
+SDK authors SHOULD provide a mechanism to distinguish between different types of errors within a custom error handler. For example, 
+an end user should have the ability to selectively ignore errors surfaced by the `SpanExporter` while logging all other errors. 
+
+### Examples 
```

Comment:
Not totally sure if this section is helpful or needed... it might not add a ton of value.

Reply:
I think this may be a case where Go is actually kind of confusing to non-gophers, but an example is always helpful.

Reply:
Might be worth converting this to Java, now that you've implemented the Java one?